specifyweb/frontend/js_src/lib/components/PickLists/definitions.ts

```diff
@@ -193,6 +193,12 @@ export const getFrontEndPickLists = f.store<{
         userTypes.map((title) => createPickListItem(title, title))
       ),
     },
+    GeologicTimePeriod: {
+      name: definePicklist('_GeologicTimePeriod', [])
+        .set('type', PickListTypes.FIELDS)
+        .set('tableName', 'geologictimeperiod')
+        .set('fieldName', 'name'),
+    },
```

Comment:
Since we want this picklist for CO -> Age, we can directly define this picklist for that relation instead of defining it for GeologicTimePeriod -> name. It would ensure we can automatically use this picklist for that relation without needing to check the field name.

Can also optionally change the picklist name from `_GeologicTimePeriod` to something alluding to EAQ so it's clearer what the picklist is for.

```suggestion
    CollectionObject: {
      age: definePicklist('_GeologicTimePeriod', [])
        .set('type', PickListTypes.FIELDS)
        .set('tableName', 'geologictimeperiod')
        .set('fieldName', 'name'),
    },
```